themes/default/content/blog/tf-migration-offer/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Terraform Migration Offer"
+date: 2023-06-12T20:05:11Z
+meta_desc: The Terraform Migration Offer provides Terraform workspace migrations as part of Pulumi Enterprise and Business Critical onboarding. 
+meta_image: meta.png
+authors:
+    - joe-duffy
+tags:
+    - terraform
+    - migration
+---
+
+Many of our users come to Pulumi from existing infrastructure as code (IaC) tools like Terraform. They choose Pulumi because of the increased developer productivity and ability to harness the complexity of the cloud when using real programming languages. However, migrating existing IaC projects to Pulumi can come with a lot of challenges and time. Today, we announced converting full Terraform projects to Pulumi directly within the Pulumi CLI. Previously, the tf2pulumi tool could convert just snippets of Terraform to Pulumi. With the release, you can now convert the vast majority of complete Terraform projects. See Justin Van Patten’s [blog post](/blog/converting-full-terraform-programs-to-pulumi) for more details.
```

Comment:
```suggestion
Many of our users come to Pulumi from existing infrastructure as code (IaC) tools like Terraform. They choose Pulumi because of the increased developer productivity and ability to harness the complexity of the cloud when using real programming languages. However, migrating existing IaC projects to Pulumi can come with a lot of challenges and time. Today, we announced converting full Terraform projects to Pulumi directly within the Pulumi CLI. Previously, the `tf2pulumi` tool could convert just snippets of Terraform to Pulumi. With the release, you can now convert the vast majority of complete Terraform projects. See Justin Van Patten’s [blog post](/blog/converting-full-terraform-programs-to-pulumi) for more details.
```